spec/spec_helper.rb

```diff
@@ -34,18 +35,21 @@ def expect_default_values(hash)
   expect(hash[SecureHeaders::XPermittedCrossDomainPolicies::HEADER_NAME]).to eq(SecureHeaders::XPermittedCrossDomainPolicies::DEFAULT_VALUE)
   expect(hash[SecureHeaders::ReferrerPolicy::HEADER_NAME]).to be_nil
   expect(hash[SecureHeaders::ExpectCertificateTransparency::HEADER_NAME]).to be_nil
+  expect(hash[SecureHeaders::ClearSiteData::HEADER_NAME]).to be_nil
+  expect(hash[SecureHeaders::ExpectCertificateTransparency::HEADER_NAME]).to be_nil
+  expect(hash[SecureHeaders::PublicKeyPins::HEADER_NAME]).to be_nil
 end
 
 module SecureHeaders
   class Configuration
     class << self
-      def clear_configurations
-        @configurations = nil
+      def clear_default_config
+        remove_instance_variable(:@default_config) if defined?(@default_config)
```

Comment:
It isn't enough to `nil` out the default config anymore since we are checking `defined?(@default_config)` when we throw an `AlreadyConfiguredError`.